Sources/SwiftSyntax/generated/Tokens.swift

```diff
@@ -14,713 +14,713 @@
+        
     )
   }
   
   public static func poundIfKeyword(
-    leadingTrivia: Trivia = [], 
-    trailingTrivia: Trivia = [], 
+    leadingTrivia: Trivia = [],
+    trailingTrivia: Trivia = [],
     presence: SourcePresence = .present
   ) -> TokenSyntax {
     return TokenSyntax(
-        .poundIfKeyword, 
-        leadingTrivia: leadingTrivia, 
-        trailingTrivia: trailingTrivia, 
-        presence: presence
-      
+      .poundIfKeyword,
+      leadingTrivia: leadingTrivia,
+      trailingTrivia: trailingTrivia,
+      presence: presence
+        
     )
   }
   
   public static func poundSourceLocationKeyword(
-    leadingTrivia: Trivia = [], 
-    trailingTrivia: Trivia = [], 
+    leadingTrivia: Trivia = [],
+    trailingTrivia: Trivia = [],
     presence: SourcePresence = .present
   ) -> TokenSyntax {
     return TokenSyntax(
-        .poundSourceLocationKeyword, 
-        leadingTrivia: leadingTrivia, 
-        trailingTrivia: trailingTrivia, 
-        presence: presence
-      
+      .poundSourceLocationKeyword,
+      leadingTrivia: leadingTrivia,
+      trailingTrivia: trailingTrivia,
+      presence: presence
+        
     )
   }
   
   public static func poundUnavailableKeyword(
-    leadingTrivia: Trivia = [], 
-    trailingTrivia: Trivia = [], 
+    leadingTrivia: Trivia = [],
+    trailingTrivia: Trivia = [],
     presence: SourcePresence = .present
   ) -> TokenSyntax {
     return TokenSyntax(
-        .poundUnavailableKeyword, 
-        leadingTrivia: leadingTrivia, 
-        trailingTrivia: trailingTrivia, 
-        presence: presence
-      
+      .poundUnavailableKeyword,
+      leadingTrivia: leadingTrivia,
+      trailingTrivia: trailingTrivia,
+      presence: presence
+        
     )
   }
   
   public static func prefixAmpersandToken(
-    leadingTrivia: Trivia = [], 
-    trailingTrivia: Trivia = [], 
+    leadingTrivia: Trivia = [],
+    trailingTrivia: Trivia = [],
     presence: SourcePresence = .present
   ) -> TokenSyntax {
     return TokenSyntax(
-        .prefixAmpersand, 
-        leadingTrivia: leadingTrivia, 
-        trailingTrivia: trailingTrivia, 
-        presence: presence
-      
+      .prefixAmpersand,
+      leadingTrivia: leadingTrivia,
+      trailingTrivia: trailingTrivia,
+      presence: presence
+        
     )
   }
   
   public static func prefixOperator(
-      _ text: String, 
-      leadingTrivia: Trivia = [], 
-      trailingTrivia: Trivia = [], 
-      presence: SourcePresence = .present
+    _ text: String,
+    leadingTrivia: Trivia = [],
+    trailingTrivia: Trivia = [],
+    presence: SourcePresence = .present
     
   ) -> TokenSyntax {
     return TokenSyntax(
-        .prefixOperator(text), 
-        leadingTrivia: leadingTrivia, 
-        trailingTrivia: trailingTrivia, 
-        presence: presence
-      
+      .prefixOperator(text),
+      leadingTrivia: leadingTrivia,
+      trailingTrivia: trailingTrivia,
+      presence: presence
+        
```

Comment:
There are multiple places with with white space.
Also think that Doug mentioned it.

Reply:
@ahoppen this one seems pretty weird to me:
1. I thought we were stripping trailing whitespace
2. We shouldn't be adding a newline here. Seems like that was broken before and after though

Reply:
This is the formatter subclass of `BasicFormat` that lives in CodeGeneration messing things up.